ds3_java_cli/src/main/java/com/spectralogic/ds3cli/util/Utils.java

```diff
@@ -303,4 +306,29 @@ public static long dateDiffToSeconds(final String diff) {
         }
         return secs;
     }
+
+    public static Properties readProperties(final String propertyFile) throws IOException {
+        final Properties props = new Properties();
+        final InputStream input = Arguments.class.getClassLoader().getResourceAsStream(propertyFile);
+        if (input != null) {
+            props.load(input);
+            return props;
+        }
+        return null;
```

Comment:
Rather than returning null, return the props object just without the properties populated.  I realize you probably didn't write this, but it would be a good change to make regardless.